src/tests/ftest/server/dynamic_start_stop.py

```diff
@@ -46,6 +45,26 @@ def verify_system_query(self):
                     member["state"], "joined",
                     "State isn't joined! Actual: {}".format(member["state"]))
 
+    def stop_server_ranks(self, ranks):
+        """Stop one or more server ranks.
+
+        Args:
+            ranks (list): [description]
+        """
+        # Stop the requested server ranks
+        ranks_str = ",".join([str(rank) for rank in ranks])
+        self.dmg_cmd.system_stop(ranks=ranks_str)
```

Comment:
FYI, I was trying to fix this test and found out that `system_stop()` fails because it's still doing the string scraping with regex. In my test run, the regex matched a single comma for some reason, so `get_numeric_list()` in general_utils.py raised AttributeError. You might want to convert `system_stop()` to use `self._get_json_result()`.

Reply:
I'm not seeing that in the multiple runs of this PR.  It is a good idea to move `DmgCommand.system_stop()` to use `self._get_json_result()`, but I don't think I'll tackle it in this PR.